Doc/library/test.rst

```diff
@@ -498,16 +500,34 @@ The :mod:`test.support` module defines the following functions:
    rather than looking directly in the path directories.
 
 
+.. function:: mark(label, *, globals=None)
```

Comment:
I would prefer to always call them "marks". The main reason is that `pytest` uses the same idea and pytest's marks are well-known.

```suggestion
.. function:: mark(name, *, globals=None)
```

Right now `mark` function adds a "label". It does not sound right.

Reply:
Or we can always call it "label".

Reply:
Yes, it is similar to pytest's markers, but there are enough differences in applying them and filtering by them. I do not have preference, "label", "marker" and "tag" are all look like synonyms in this context to me.